backend/btrixcloud/models.py

```diff
@@ -337,6 +337,7 @@ class RawCrawlConfig(BaseModel):
     userAgent: Optional[str] = None
 
     selectLinks: List[str] = ["a[href]->href"]
+    clickSelector: str = "a"
```

Comment:
Could we name this more explicitly `autoclickSelector`, since it's specific to autoclick?

Reply:
It needs to be named the same as the crawler CLI since the options here get automatically converted into CLI flags. We could add `autoclickSelector` as an alias in the crawler but it's currently named `clickSelector`